pkgs/applications/science/robotics/emuflight-configurator/default.nix

```diff
@@ -0,0 +1,50 @@
+{lib, stdenv, fetchurl, unzip, makeDesktopItem, nwjs, wrapGAppsHook, gsettings-desktop-schemas, gtk3 }:
+
+let
+  pname = "emuflight-configurator";
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = pname;
+    icon = pname;
+    comment = "Emuflight configuration tool";
+    desktopName = "Emuflight Configurator";
+    genericName = "Flight controller configuration tool";
+  };
+in
+stdenv.mkDerivation rec {
+  inherit pname;
+  version = "0.3.5";
+  src = fetchurl {
+    url = "https://github.com/emuflight/EmuConfigurator/releases/download/${version}/emuflight-configurator_${version}_linux64.zip";
+    sha256 = "d55bdc52cf93d58c728ccb296ef912a5fc0f42c57ed95f3ded5f85d1c10838c4";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook unzip ];
+
+  buildInputs = [ gsettings-desktop-schemas gtk3 ];
+
+  installPhase = ''
+    mkdir -p $out/bin \
+             $out/opt/${pname}
+
+    cp -r . $out/opt/${pname}/
+    install -m 444 -D icon/emu_icon_128.png $out/share/icons/hicolor/128x128/apps/${pname}.png
+    cp -r ${desktopItem}/share/applications $out/share/
```

Comment:
Please use copyDeskopItems like https://github.com/NixOS/nixpkgs/blob/7f5807d5af9f6ec2749b2c16302d9bbeed370bbf/pkgs/applications/graphics/openboard/default.nix#L75-L86.

Reply:
I am not sure if I follow this one @SuperSandro2000.  What is wrong with the current desktopItem?  Should I not be copying it into share?

Reply:
desktopItems together with copyDesktopItems copies the file automatically.

Reply:
aah, I always get confused by buildInputs that do things rather than just provide a dependency